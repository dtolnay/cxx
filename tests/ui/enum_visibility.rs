#[cxx::bridge]
mod ffi {
    enum Bad {
        A(pub i32),
        B(bool),
    }
}

fn main() {}
