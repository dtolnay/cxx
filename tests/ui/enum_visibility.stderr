error[E0449]: visibility qualifiers are not permitted here
 --> tests/ui/enum_visibility.rs:4:11
  |
4 |         A(pub i32),
  |           ^^^
  |
  = note: enum variants and their fields always share the visibility of the enum they are in
